### DISCUSSION TITLE
Repository Projects

### DISCUSSION THREAD

**Discussion by gaganyadav80** [pd4d10/git-touch#215] [Ideas]
It would be really nice to have github repository projects in this app. It is a very handy tool and it would be more amazing if we could add, edit, and manage it on this app.

**Comment by luni-moon** [comment]
I would love to see that added as well!

**Comment by gaganyadav80** [comment]
@shreyas1599 what do you think?

**Reply by shreyas1599** [reply]
Yeah I thought of working on this a while back but haven't found the time. Might take a while. Are you interested in doing it?

**Reply by gaganyadav80** [reply]
I'm interested but busy with few of my own project. I will try to do it in my free time.
